test/parallel/test-fs-rmdir-recursive-warns-not-found.js

```diff
@@ -6,7 +6,6 @@ const path = require('path');
 
 tmpdir.refresh();
 
-
```

Comment:
nit: we usually try to avoid unrelated whitespace changes.

Reply:
Good to know. I'm pretty sure I accidentally added this extra whitespace in my previous PR so I was just cleaning it up.